### PR TITLE
Fix linting errors in standard 9.0

### PR DESCRIPTION
**Pull request by timneutkens**
Thanks to @aranajhonny for pointing out there was an error in the build

### CODE REVIEW

File: server/on-demand-entry-handler.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import { parse } from 'url'
 import resolvePath from './resolve'
 import touch from 'touch'
 
-const ADDED = Symbol()
-const BUILDING = Symbol()
-const BUILT = Symbol()
+const ADDED = Symbol('added')
+const BUILDING = Symbol('building')
+const BUILT = Symbol('built')
```
Comment (timneutkens) [inline server/on-demand-entry-handler.js]: This is just a description for debugging purposes. See https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Symbol

File: examples/with-custom-babel-config/pages/index.js [hunk 2]
```diff
@@ -27,10 +27,13 @@ export default class MyLuckNo extends React.Component {
     // using babel-preset-stage-0
     const message = do {
       if (randomNo < 30) {
+        // eslint-disable-next-line no-unused-expressions
         'Do not give up. Try again.'
       } else if (randomNo < 60) {
+        // eslint-disable-next-line no-unused-expressions
         'You are a lucky guy'
       } else {
+        // eslint-disable-next-line no-unused-expressions
```
Comment (timneutkens) [inline examples/with-custom-babel-config/pages/index.js]: Stage-0 feature is used. So we have to disable style checking for that.

**Comment by rauchg** [comment]
I'd love to migrate to prettier + eslint:recommended

**Comment by timneutkens** [comment]
Okay, i'll put it on my list!

**Pull request merged**